src/supermin-init-prelude.sh

```diff
@@ -0,0 +1,27 @@
+mount -t proc /proc /proc
+mount -t sysfs /sys /sys
+mount -t devtmpfs devtmpfs /dev
+
+# load selinux policy
+LANG=C /sbin/load_policy  -i
+
+# load kernel module for 9pnet_virtio for 9pfs mount
+/sbin/modprobe 9pnet_virtio
+
+# need fuse module for rofiles-fuse/bwrap during post scripts run
+/sbin/modprobe fuse
+
+# set up networking
+/usr/sbin/dhclient eth0
```

Comment:
Do we actually need this now?

Reply:
i think so. we run the compose in the VM and it pulls rpms from remote yum repos right?